projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -384,7 +384,8 @@ private function set_token_cookie( $token ) {
 		}
 
 		if ( ! empty( $token ) && false === headers_sent() ) {
-			setcookie( self::JWT_AUTH_TOKEN_COOKIE_NAME, $token, 0, '/', COOKIE_DOMAIN, is_ssl(), true ); // httponly -- used by visitor_can_view_content() within the PHP context.
+			// Set the token to expire in one week to prevent ever-lasting subscriptions (if cancelled) and to force user to reload
```

Comment:
I think we should set the expire date at the token level instead of here, so if the token is too old we can require to log-in to get a new one.